src/zeit/cms/content/tests/test_metadata.py

```diff
@@ -35,3 +37,15 @@ def test_channels_already_set_does_not_change_anything(self):
                     zeit.cms.testcontenttype.interfaces.ITestContentType,
                     'ressort'))
             self.assertEqual((('Deutschland', None),), co.channels)
+
+    def test_channels_are_not_set_if_product_forbids_it(self):
+        article = TestContentType()
+        article.product = zeit.cms.content.sources.Product(u'ZEI')
+        self.repository['testcontent'] = article
+        with checked_out(self.repository['testcontent']) as co:
+            co.ressort = u'Deutschland'
+            zope.lifecycleevent.modified(
+                co, zope.lifecycleevent.Attributes(
+                    zeit.cms.testcontenttype.interfaces.ITestContentType,
+                    'ressort'))
+            self.assertEqual((), co.channels)
```

Comment:
Gibt es irgendwo den positiven Test, dass ein Channel automatisch vergeben wird? In der Test-Klasse geht es ja eher ums kopieren.